server/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+edition = "2021"            # Allowed values are 2015, 2018, 2021
+name    = "svc-scheduler"
+version = "0.2.5-develop.3"
```

Comment:
Please reset the version to "0.1.0" so the release workflow won't run into issues and we get a proper CHANGELOG generated